### ISSUE TITLE
Add decibel measurment (Peak Signal-to-noise Ratio)

### ISSUE THREAD

**Issue by cMancio00** [cMancio00/Super-Resolution#12]
It is common to use PSNR for measure the quality of a predicted image over the real one.
A library  that provides this measurement is [Lightning](https://lightning.ai/docs/torchmetrics/stable/image/peak_signal_noise_ratio.html)

- [x] Add the measurement in the training loop
- [x] Print the average PSNR next to the average L1 loss
- [x] Save the metric on a csv with L1 loss
- [x] Update requirements with Lightining
